ui/lib/kv/addon/components/kv-patch/editor/form.js

```diff
@@ -72,10 +72,10 @@ export class KeyValueState {
   }
 }
 
-export default class KvPatchEditor extends Component {
+export default class KvPatchEditorForm extends Component {
   @tracked patchData; // key value pairs in form
   @tracked showSubkeys = false;
-  @tracked submitError;
```

Comment:
`@submitError` now just comes from the parent if there's a problem making the request, internally we just track `validationError`